site/content/tutorial/02-reactivity/04-updating-arrays-and-objects/text.md

```diff
@@ -2,7 +2,7 @@
 title: Updating arrays and objects
 ---
 
-Because Svelte's reactivity is triggered by assignments, using array methods like `push` and `splice` won't automatically cause updates. For example, clicking the button doesn't do anything.
+Because Svelte's reactivity is triggered by assignments, using array methods like `push` and `splice` won't automatically cause updates. For example, currently clicking the button doesn't do anything.
```

Comment:
I like the previous version better, "currently" doesn't add any value for me here.